src/index.html

```diff
@@ -1,17 +1,183 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()"
+    class="form"
+    >
+      <fieldset class="firstform" id="first1">
+        <legend>Personal information</legend>
+        <div class="label">
+          <label for="sname">
+            Surname:
+          </label>
+          <input
+            type="text"
+            id="sname"
+            name="fname"
+            required
+            autocomplete="off"
+          >
+        </div>
+        <div class="label">
+          <label for="fname">Name:</label>
+          <input
+            type="text"
+            id="fname"
+            name="fname"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div class="label">
+          <label for="old">How old are You?</label>
+          <input
+            type="number"
+            value="12"
+            maxlength="100"
+            id="old"
+            name="fname"
+            required
+            min="1"
+            max="199"
+          >
+        </div>
+        <div class="label">
+          <label for="datebirth">Full date of birth:</label>
+          <input
+            type="date"
+            id="datebirth"
+            name="fname"
+            required
+          >
+        </div>
+        <div class="label">
+          <label for="check">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="check"
+            name="terms"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="firstform" id="first2">
+        <legend>Registration</legend>
+        <div class="label">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            required
+            multiple
+            placeholder="email@example.com"
+            autocomplete="off"
+          >
+        </div>
+        <div class="label">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            id="pass"
+            name="password"
+            required
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="firstform" id="first3">
+        <legend>An interesting fact about you!</legend>
+        <div class="label">
+          <label for="cats">Do you love cats?</label>
+          <input
+            type="radio"
+            id="cats"
+            name="fav_language"
+            value="yes"
+          >
+            Yes
+          <input
+            type="radio"
+            id="cate"
```

Comment:
```suggestion
            id="cate"
```
cate?
These two radios should have ids 'yes' and 'no', for example